modules/cluster-logging-release-notes-5.3.z.adoc

```diff
@@ -0,0 +1,132 @@
+//Z-stream Release Notes by Version
+
+[id="cluster-logging-release-notes-5-3-3"]
+= OpenShift Logging 5.3.3
+This release includes link:https://access.redhat.com/errata/RHSA-2022:0227[RHSA-2022:0227 OpenShift Logging Bug Fix Release 5.3.3]
+
+[id="openshift-logging-5-3-3-bug-fixes"]
+== Bug fixes
+* Before this update, changes to the metrics dashboards had not yet been deployed because the cluster-logging-operator did not correctly compare existing and desired configmaps containing the dashboard.  This update fixes the logic by adding a the dashboard unique hash value to the object labels.(link:https://issues.redhat.com/browse/LOG-2066[LOG-2066])
+
+* This update changes the log4j dependency to 2.17.1 to resolve link:https://access.redhat.com/security/cve/CVE-2021-44832[CVE-2021-44832].(link:https://issues.redhat.com/browse/LOG-2102[LOG-2102])
+
+== CVEs
+[id="openshift-logging-5-3-3-CVEs"]
+.Click to expand CVEs
+[%collapsible]
+====
+* link:https://access.redhat.com/security/cve/CVE-2021-27292[CVE-2021-27292]
+** link:https://bugzilla.redhat.com/show_bug.cgi?id=1940613[BZ-1940613]
+* link:https://access.redhat.com/security/cve/CVE-2021-44832[CVE-2021-44832]
+** link:https://bugzilla.redhat.com/show_bug.cgi?id=2035951[BZ-2035951]
+====
+
+[id="cluster-logging-release-notes-5-3-2"]
+= OpenShift Logging 5.3.2
+This release includes link:https://access.redhat.com/errata/RHSA-2022:0044[RHSA-2022:0044 OpenShift Logging Bug Fix Release 5.3.2]
+
+[id="openshift-logging-5-3-2-bug-fixes"]
+== Bug fixes
+* Before this update, Elasticsearch rejected logs from the eventrouter due to a parsing error. This update changes the data model to resolve the parsing error. However, as a result, existing indices will cause an warning/error within Kibana. The field ("kubernetes.event.metadata.resourceVersion") will cause an error until the removal or reindexing of existing indices. If this field is not in use in Kibana, then you can ignore the error message. For those with a delete policy, that policy will eventually remove old indices. When that occurs, the message will no longer generate. Those without a delete policy in place will need to manually reindex in order to remove the message.(link:https://issues.redhat.com/browse/LOG-2088[LOG-2088])
+
+* Before this update, the OpenShift Logging Dashboard displayed the wrong pod namespace in the table which displays top producing and collected containers over the last 24 hours. With this update, the OpenShift Logging Dashboard displays the correct pod namespace.(link:https://issues.redhat.com/browse/LOG-2051[LOG-2051])
```

Comment:
Does the following guidance from the IBM SG apply here:
"Use that, without a comma, to introduce a restrictive clause. A restrictive clause is essential to the
meaning of the sentence. If the restrictive clause is removed, the meaning of the sentence changes."
If so, consider:
```suggestion
* Before this update, the OpenShift Logging Dashboard displayed the wrong pod namespace in the table that displays top producing and collected containers over the last 24 hours. With this update, the OpenShift Logging Dashboard displays the correct pod namespace.(link:https://issues.redhat.com/browse/LOG-2051[LOG-2051])
```